### PR TITLE
Port status server to aiohttp

### PR DESCRIPTION
**Pull request by godwhoa**
This branch is based off of pypng, hopefully diffs will be clean once that is merged.

### CODE REVIEW

File: piqueserver/statusserver.py [hunk 1]
```diff
@@ -1,144 +1,153 @@
-# Copyright (c) junk/someonesomewhere 2011.
```
Comment (NotAFile) [inline piqueserver/statusserver.py]: pretty sure this is still legally a derived work,  this copyright notice can't be removed. I hate these long headers too, but people are torn on if just putting a LICENSE file in your folder is legally sufficient, and the GPL license recommends adding one. Here's some options:

 - put the licence preamble and copyright info at the end of the file
 - use a spdx ID, which is just one line. I'm not entirely sure if this is enough legally, but it's better than no copyright notice and also machine readable. SPDX recommands addng the copyright header even when you're using SPDX.

File: piqueserver/server.py [hunk 2]
```diff
@@ -378,15 +377,16 @@ def __init__(self, interface: bytes, config_dict: Dict[str, Any]) -> None:
...
+            self.status_server = StatusServer(self)
+            ensureDeferred(self.status_server.listen())
         if ban_publish.get():
             from piqueserver.banpublish import PublishServer
             self.ban_publish = PublishServer(self, ban_publish_port.get())
         if bans_urls.get():
             from piqueserver import bansubscribe
             self.ban_manager = bansubscribe.BanManager(self)
-        self.start_time = reactor.seconds()
+        self.start_time = time.time()
```
[the remark this answers is not in the document]
Comment (godwhoa) [reply piqueserver/server.py]: Used `time.time` in the status server so switched it here. Also realised `reactor.seconds` is just an alias for `time.time`.

[the remark this answers is not in the document]
Comment (NotAFile) [reply piqueserver/server.py]: yeah, let's hope it stays that way so things don't break. Finding that out from the code was extremely convoluted, wow.

File: piqueserver/statusserver.py [hunk 3]
```diff
@@ -15,130 +15,150 @@
...
-from twisted.web.resource import Resource
-from piqueserver.config import config
-import piqueserver.web
+import json
+import time
+import png
+from io import BytesIO
+from aiohttp.abc import AbstractAccessLogger
+from twisted.logger import Logger
+from piqueserver.utils import as_deferred, as_future
```
Comment (NotAFile) [inline piqueserver/statusserver.py]:
```suggestion
from piqueserver.utils import as_deferred
```

unused import

**Pull request merged**